test/Etg.Yams.Core.Test/Process/EnvironmentTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Etg.Yams.Utils;
+using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+using Xunit;
+
+namespace Etg.Yams.Test.Process
+{
+    public class EnvironmentFixture : IDisposable
+    {
+        readonly string originalMachinePath;
+        public EnvironmentFixture()
+        {
+            originalMachinePath = Environment.GetEnvironmentVariable("PATH", EnvironmentVariableTarget.Machine);
+        }
+
+        public void Dispose()
+        {
+            try
+            {
+                Environment.SetEnvironmentVariable("PATH", originalMachinePath, EnvironmentVariableTarget.Machine);
```

Comment:
I'd rather not modify the path in a test. What happens if tests run in parallel?